onnx/defs/nn/defs.cc

```diff
@@ -3057,6 +3059,8 @@ def compute_rotary_embedding(
     output = np.concatenate((x_rotate, x_not_rotate), axis=-1)
     if len(input.shape) == 3:
         output = np.reshape(output, input.shape)
+    elif len(original_input_shape) == 4:
```

Comment:
minor nit: should we just make this an `else`? Or, use similar conditions in both then and else, it is unclear why one uses `input.shape` and the other uses `original_input_shape`

Reply:
Fixed, made it else for now. I think the actual ref impl, uses original_input_shape in both branches